### ISSUE TITLE
astro.build main page performance is awful in Safari

### ISSUE THREAD

**Issue by ermakovich** [withastro/astro.build#1274]
There are severe performance issues in Safari on the main page. Most likely because of expensive relayout involving too many SVG elements on the page. Hover animations, which are causing relayout, are taking up to 1 second.

Safari 17.5
MacBook Pro 2019 2.6Hz 6-Core Intel Core i7

**Comment by delucis** [comment]
Thanks for the report @ermakovich! Could you provide a bit more detail? Are there particular parts of the page that have issues for example? A screenshot or two would be amazing. I've tested in Safari on a 2019 MacBook but couldn't immediately reproduce the issues.

**Comment by ermakovich** [comment]
@delucis Unfortunately I can't say which particular part contributes the most to the performance issues. My initial assumption regarding SVG elements was wrong. I tried to remove all SVG elements (around 200) from the page using script, and it didn't have any notable effect. 

Attaching Safari timeline export file here: 
[astro.build-recording.json](https://github.com/user-attachments/files/17173633/astro.build-recording.json)

It can be imported on Safari Web Inpector Timelines tab. In the scenario for this timeline I'm doing simple mouse over a button in "Zero Lock-In" block.

Also attaching a screenshot:
<img width="793" alt="Screenshot 2024-09-28 at 10 54 16" src="https://github.com/user-attachments/assets/8cbf2700-ccff-471e-9975-7b7add9e8cf0">

It can be seen that one of repaints during this event takes 915ms.

As a side note I need to say that in Chrome there are no performance issues at all.

**Comment by delucis** [comment]
Sorry for the delay getting back to you @ermakovich and thank you very much for the timeline export. I still haven’t been able to reproduce unfortunately and all I can see in the export is that Safari is spending a _suprisingly_ long time painting (3 seconds is a long time!)

My guess is that this is to do with `blur` filters. I discovered two elements that were using a 160px blur as a background decoration, including one that is visible in the screenshots of your timeline.

Although I can’t really verify this is a fix on my copies of Safari, I opened #1350 to address this.

**Comment by ermakovich** [comment]
@delucis hi @delucis , great find with large blurs! I think I initially read Safari timings wrong, the line I selected in the screenshot was actually taking a fraction of a millisecond rather than 915ms :) Sorry for some possible confusion. And 3 seconds you mentioned are actually not durations, but rather a Start Time in a timeline.

Anyway, I tested the website today, and I can confirm that it works like a charm now! 👍
